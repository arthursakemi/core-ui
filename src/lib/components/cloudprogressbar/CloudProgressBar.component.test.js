import initStoryshots from "@storybook/addon-storyshots";

initStoryshots({
  configPath: "src/lib/components/cloudprogressbar/__snapshots__"
});
