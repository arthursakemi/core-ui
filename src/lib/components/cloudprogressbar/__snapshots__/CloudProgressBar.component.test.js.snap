// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots CloudProgressBar Default 1`] = `
<div
  className="sc-htpNat iharJH"
>
  <h3
    className="sc-bxivhb iDJTyr"
  >
    CloudProgressBar
  </h3>
  <div
    style={
      Object {
        "margin": "20px 0 20px 0",
        "width": "300px",
      }
    }
  >
    <div
      className="sc-bdVaJa jjzlwn sc-cloudprogressbar"
    >
      <svg
        enable-background="new 0 0 72 72"
        viewBox="0 15 72 42"
        xmlns="http://www.w3.org/2000/svg"
      >
        <g>
          <path
            d="M15.9,30.3c0,0.4-0.4,0.8-0.8,0.8C10,31.6,6,36.7,6,42.9c0,6.6,4.5,11.9,10.2,11.9h38.7C61,54.8,66,49.1,66,42.2c0-6.6-4.6-12.1-10.4-12.5c-0.4,0-0.8-0.3-0.9-0.8c-1.3-6.7-7.3-11.7-14.3-11.7c-4.6,0-8.7,2.1-11.3,5.4C28.8,23,28.3,23.1,28,23c-1-0.4-2.1-0.6-3.3-0.6C20.1,22.3,16.3,25.8,15.9,30.3z"
            fill="none"
            stroke="blue"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-miterlimit="10"
            stroke-width="1.5px"
          />
        </g>
      </svg>
      <div
        className="sc-bwzfXH fiImRm"
      >
        <svg
          enable-background="new 0 0 72 72"
          viewBox="0 15 72 42"
          xmlns="http://www.w3.org/2000/svg"
        >
          <g>
            <path
              d="M15.9,30.3c0,0.4-0.4,0.8-0.8,0.8C10,31.6,6,36.7,6,42.9c0,6.6,4.5,11.9,10.2,11.9h38.7C61,54.8,66,49.1,66,42.2c0-6.6-4.6-12.1-10.4-12.5c-0.4,0-0.8-0.3-0.9-0.8c-1.3-6.7-7.3-11.7-14.3-11.7c-4.6,0-8.7,2.1-11.3,5.4C28.8,23,28.3,23.1,28,23c-1-0.4-2.1-0.6-3.3-0.6C20.1,22.3,16.3,25.8,15.9,30.3z"
              fill="none"
              stroke="yellow"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-miterlimit="10"
              stroke-width="1.5px"
            />
          </g>
        </svg>
      </div>
      <div
        style={
          Object {
            "left": "50%",
            "position": "absolute",
            "top": "50%",
            "transform": "translate(-50%,-50%)",
          }
        }
      >
        <h3
          className="sc-bxivhb iDJTyr"
        >
          30%
        </h3>
        <span
          className="sc-ifAKCX byxrvZ"
        >
          of the quotas has been reached.
        </span>
      </div>
    </div>
  </div>
  <div
    style={
      Object {
        "margin": "20px 0 20px 0",
        "width": "400px",
      }
    }
  >
    <div
      className="sc-bdVaJa jjzlwn sc-cloudprogressbar"
    >
      <svg
        enable-background="new 0 0 72 72"
        viewBox="0 15 72 42"
        xmlns="http://www.w3.org/2000/svg"
      >
        <g>
          <path
            d="M15.9,30.3c0,0.4-0.4,0.8-0.8,0.8C10,31.6,6,36.7,6,42.9c0,6.6,4.5,11.9,10.2,11.9h38.7C61,54.8,66,49.1,66,42.2c0-6.6-4.6-12.1-10.4-12.5c-0.4,0-0.8-0.3-0.9-0.8c-1.3-6.7-7.3-11.7-14.3-11.7c-4.6,0-8.7,2.1-11.3,5.4C28.8,23,28.3,23.1,28,23c-1-0.4-2.1-0.6-3.3-0.6C20.1,22.3,16.3,25.8,15.9,30.3z"
            fill="none"
            stroke="#000"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-miterlimit="10"
            stroke-width="1.5px"
          />
        </g>
      </svg>
      <div
        className="sc-bwzfXH cLuCbL"
      >
        <svg
          enable-background="new 0 0 72 72"
          viewBox="0 15 72 42"
          xmlns="http://www.w3.org/2000/svg"
        >
          <g>
            <path
              d="M15.9,30.3c0,0.4-0.4,0.8-0.8,0.8C10,31.6,6,36.7,6,42.9c0,6.6,4.5,11.9,10.2,11.9h38.7C61,54.8,66,49.1,66,42.2c0-6.6-4.6-12.1-10.4-12.5c-0.4,0-0.8-0.3-0.9-0.8c-1.3-6.7-7.3-11.7-14.3-11.7c-4.6,0-8.7,2.1-11.3,5.4C28.8,23,28.3,23.1,28,23c-1-0.4-2.1-0.6-3.3-0.6C20.1,22.3,16.3,25.8,15.9,30.3z"
              fill="none"
              stroke="#ccc"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-miterlimit="10"
              stroke-width="1.5px"
            />
          </g>
        </svg>
      </div>
      <div
        style={
          Object {
            "left": "50%",
            "position": "absolute",
            "top": "50%",
            "transform": "translate(-50%,-50%)",
          }
        }
      >
        <h3
          className="sc-bxivhb iDJTyr"
        >
          50%
        </h3>
        <span
          className="sc-ifAKCX byxrvZ"
        >
          of the quotas has been reached
        </span>
      </div>
    </div>
  </div>
  <div
    style={
      Object {
        "margin": "20px 0 20px 0",
        "width": "500px",
      }
    }
  >
    <div
      className="sc-bdVaJa jjzlwn sc-cloudprogressbar"
    >
      <svg
        enable-background="new 0 0 72 72"
        viewBox="0 15 72 42"
        xmlns="http://www.w3.org/2000/svg"
      >
        <g>
          <path
            d="M15.9,30.3c0,0.4-0.4,0.8-0.8,0.8C10,31.6,6,36.7,6,42.9c0,6.6,4.5,11.9,10.2,11.9h38.7C61,54.8,66,49.1,66,42.2c0-6.6-4.6-12.1-10.4-12.5c-0.4,0-0.8-0.3-0.9-0.8c-1.3-6.7-7.3-11.7-14.3-11.7c-4.6,0-8.7,2.1-11.3,5.4C28.8,23,28.3,23.1,28,23c-1-0.4-2.1-0.6-3.3-0.6C20.1,22.3,16.3,25.8,15.9,30.3z"
            fill="none"
            stroke="#83d136"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-miterlimit="10"
            stroke-width="2px"
          />
        </g>
      </svg>
      <div
        className="sc-bwzfXH hnLSfV"
      >
        <svg
          enable-background="new 0 0 72 72"
          viewBox="0 15 72 42"
          xmlns="http://www.w3.org/2000/svg"
        >
          <g>
            <path
              d="M15.9,30.3c0,0.4-0.4,0.8-0.8,0.8C10,31.6,6,36.7,6,42.9c0,6.6,4.5,11.9,10.2,11.9h38.7C61,54.8,66,49.1,66,42.2c0-6.6-4.6-12.1-10.4-12.5c-0.4,0-0.8-0.3-0.9-0.8c-1.3-6.7-7.3-11.7-14.3-11.7c-4.6,0-8.7,2.1-11.3,5.4C28.8,23,28.3,23.1,28,23c-1-0.4-2.1-0.6-3.3-0.6C20.1,22.3,16.3,25.8,15.9,30.3z"
              fill="none"
              stroke="#3d383a"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-miterlimit="10"
              stroke-width="2px"
            />
          </g>
        </svg>
      </div>
      <div
        style={
          Object {
            "left": "50%",
            "position": "absolute",
            "top": "50%",
            "transform": "translate(-50%,-50%)",
          }
        }
      >
        <h3
          className="sc-bxivhb iDJTyr"
        >
          80%
        </h3>
        <span
          className="sc-ifAKCX byxrvZ"
        >
          of the quotas has been reached
        </span>
      </div>
    </div>
  </div>
  <div
    style={
      Object {
        "margin": "20px 0 20px 0",
        "width": "600px",
      }
    }
  >
    <div
      className="sc-bdVaJa jjzlwn sc-cloudprogressbar"
    >
      <svg
        enable-background="new 0 0 72 72"
        viewBox="0 15 72 42"
        xmlns="http://www.w3.org/2000/svg"
      >
        <g>
          <path
            d="M15.9,30.3c0,0.4-0.4,0.8-0.8,0.8C10,31.6,6,36.7,6,42.9c0,6.6,4.5,11.9,10.2,11.9h38.7C61,54.8,66,49.1,66,42.2c0-6.6-4.6-12.1-10.4-12.5c-0.4,0-0.8-0.3-0.9-0.8c-1.3-6.7-7.3-11.7-14.3-11.7c-4.6,0-8.7,2.1-11.3,5.4C28.8,23,28.3,23.1,28,23c-1-0.4-2.1-0.6-3.3-0.6C20.1,22.3,16.3,25.8,15.9,30.3z"
            fill="none"
            stroke="#A3FFCB"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-miterlimit="10"
            stroke-width="2px"
          />
        </g>
      </svg>
      <div
        className="sc-bwzfXH iQuCir"
      >
        <svg
          enable-background="new 0 0 72 72"
          viewBox="0 15 72 42"
          xmlns="http://www.w3.org/2000/svg"
        >
          <g>
            <path
              d="M15.9,30.3c0,0.4-0.4,0.8-0.8,0.8C10,31.6,6,36.7,6,42.9c0,6.6,4.5,11.9,10.2,11.9h38.7C61,54.8,66,49.1,66,42.2c0-6.6-4.6-12.1-10.4-12.5c-0.4,0-0.8-0.3-0.9-0.8c-1.3-6.7-7.3-11.7-14.3-11.7c-4.6,0-8.7,2.1-11.3,5.4C28.8,23,28.3,23.1,28,23c-1-0.4-2.1-0.6-3.3-0.6C20.1,22.3,16.3,25.8,15.9,30.3z"
              fill="none"
              stroke="#BDFFB0"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-miterlimit="10"
              stroke-width="2px"
            />
          </g>
        </svg>
      </div>
      <div
        style={
          Object {
            "left": "50%",
            "position": "absolute",
            "top": "50%",
            "transform": "translate(-50%,-50%)",
          }
        }
      >
        <h3
          className="sc-bxivhb iDJTyr"
        >
          70%
        </h3>
        <span
          className="sc-ifAKCX byxrvZ"
        >
          of the quotas has been reached
        </span>
      </div>
    </div>
  </div>
</div>
`;
